lib/config/definitions.ts

```diff
@@ -1217,8 +1217,14 @@ const options: RenovateOptions[] = [
     name: 'prCreation',
     description: 'When to create the PR for a branch.',
     type: 'string',
-    allowedValues: ['immediate', 'not-pending', 'status-success', 'approval'],
-    default: 'immediate',
+    allowedValues: [
+      'auto',
+      'immediate',
+      'not-pending',
+      'status-success',
+      'approval',
+    ],
+    default: 'auto',
```

Comment:
Seems we're changing the default branch creation time from `immediate` to `auto`? Is that intended here?

Does this change anything for users with a default Renovate bot config?

Reply:
The idea is that auto=immediate unless rollupName is defined. Therefore it's backwards compatible

Reply:
So roll-up or need approval by default?